src/libtriton/bindings/python/objects/pyTritonContext.cpp

```diff
@@ -3407,7 +3428,7 @@ namespace triton {
         {"newSymbolicVariable",                 (PyCFunction)TritonContext_newSymbolicVariable,                         METH_VARARGS,                  ""},
         {"popPathConstraint",                   (PyCFunction)TritonContext_popPathConstraint,                           METH_NOARGS,                   ""},
         {"processing",                          (PyCFunction)TritonContext_processing,                                  METH_O,                        ""},
-        {"pushPathConstraint",                  (PyCFunction)TritonContext_pushPathConstraint,                          METH_O,                        ""},
+        {"pushPathConstraint",                  (PyCFunction)TritonContext_pushPathConstraint,                          METH_VARARGS | METH_KEYWORDS,  ""},
```

Comment:
Can you cast like this (to avoid warning in some Python versions when compiling):

`(PyCFunction)(void*)(PyCFunctionWithKeywords)TritonContext_pushPathConstraint`